src/js/components/molecules/wizard-step-download-list/wizard-step-download-list.vue.html

```diff
@@ -1,5 +1,6 @@
-<form class="wizard-step-download-list" method="get" :action="file">
-    <HeaderBlock  class="wizard-list-download-list__header" :title="title" :subtitle="subtitle" />
+<form class="wizard-step-download-list" target="_blank" method="get" :action="file" @submit.prevent="handleSubmit">
+    <HeaderBlock class="wizard-step-download-list__header" :title="title" :subtitle="subtitle" />
+    <p class="wizard-step-download-list__description flexgrid__cell--sm-3" v-if="description">{{description}}</p>
     <ul class="wizard-step-download-list__items flexgrid flexgrid--mobile-xl-row">
         <li class="wizard-step-download-list__item flexgrid__cell--sm-3" v-if="items" v-for="(item, index) in items">
```

Comment:
Not this PR but the `v-if` and `v-for` on the same element is not recommended. See https://vuejs.org/v2/style-guide/#Avoid-v-if-with-v-for-essential